package.json

```diff
@@ -124,6 +124,7 @@
     "@storybook/manager-webpack5": "6.5.15",
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 544 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26504&commitId=ef59ca7577e910f30ad7c565e034b5a05434ba35&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1235 screenshots unchanged, 0 screenshots with different dimensions and 544 screenshots with visible difference.


  <details><summary><b>unknown</b> 544 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion Converged.disabled - Dark Mode.normal.chromium.png|31583| Changed|
|Accordion Converged.disabled - High Contrast.normal.chromium.png|37325| Changed|
|Accordion Converged.disabled.normal.chromium.png|1526| Changed|
|Accordion Converged.expandIcon   Icon-   - Dark Mode.normal.chromium.png|31478| Changed|
|Accordion Converged.expandIcon   Icon-   - High Contrast.normal.chromium.png|37507| Changed|
|Accordion Converged.expandIcon   Icon-   - RTL.normal.chromium.png|2174| Changed|
|Accordion Converged.expandIcon   Icon-  .normal.chromium.png|1571| Changed|
|Accordion Converged.expandIconPosition  end  - RTL.normal.chromium.png|1655| Changed|
|Accordion Converged.expandIconPosition  end .normal.chromium.png|1365| Changed|
|Accordion Converged.icon   Icon-   - RTL.normal.chromium.png|1429| Changed|
|Accordion Converged.icon   Icon-  .normal.chromium.png|1122| Changed|
|Accordion Converged.size - RTL.normal.chromium.png|5665| Changed|
|Accordion Converged.size.normal.chromium.png|5044| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus closed.chromium.png|25497| Changed|
|Accordion Converged.visibility+focus - Dark Mode.focus opened.chromium.png|25813| Changed|
|Accordion Converged.visibility+focus - Dark Mode.normal.chromium.png|25813| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus closed.chromium.png|30400| Changed|
|Accordion Converged.visibility+focus - High Contrast.focus opened.chromium.png|30659| Changed|
|Accordion Converged.visibility+focus - High Contrast.normal.chromium.png|30659| Changed|
|Accordion Converged.visibility+focus - RTL.focus closed.chromium.png|2099| Changed|
|Accordion Converged.visibility+focus - RTL.focus opened.chromium.png|1722| Changed|
|Accordion Converged.visibility+focus - RTL.normal.chromium.png|1722| Changed|
|Accordion Converged.visibility+focus.focus closed.chromium.png|1835| Changed|
|Accordion Converged.visibility+focus.focus opened.chromium.png|1457| Changed|
|Accordion Converged.visibility+focus.normal.chromium.png|1457| Changed|
|Badge Converged.filled - Dark Mode.chromium.png|364789| Changed|
|Badge Converged.filled - High Contrast.chromium.png|434069| Changed|
|Badge Converged.filled.chromium.png|20585| Changed|
|Badge Converged.ghost - Dark Mode.chromium.png|373319| Changed|
|Badge Converged.ghost - High Contrast.chromium.png|446103| Changed|
|Badge Converged.ghost.chromium.png|11017| Changed|
|Badge Converged.outline - Dark Mode.chromium.png|362951| Changed|
|Badge Converged.outline - High Contrast.chromium.png|430944| Changed|
|Badge Converged.outline.chromium.png|25950| Changed|
|Badge Converged.size- extra-large - RTL.chromium.png|56162| Changed|
|Badge Converged.size- extra-large.chromium.png|44818| Changed|
|Badge Converged.size- extra-small - RTL.chromium.png|4631| Changed|
|Badge Converged.size- extra-small.chromium.png|4243| Changed|
|Badge Converged.size- large - RTL.chromium.png|45063| Changed|
|Badge Converged.size- large.chromium.png|35611| Changed|
|Badge Converged.size- medium - RTL.chromium.png|39680| Changed|
|Badge Converged.size- medium.chromium.png|31219| Changed|
|Badge Converged.size- small - RTL.chromium.png|33490| Changed|
|Badge Converged.size- small.chromium.png|26508| Changed|
|Badge Converged.size- tiny - RTL.chromium.png|3056| Changed|
|Badge Converged.size- tiny.chromium.png|2833| Changed|
|Badge Converged.tint - Dark Mode.chromium.png|368422| Changed|
|Badge Converged.tint - High Contrast.chromium.png|437240| Changed|
|Badge Converged.tint.chromium.png|17543| Changed|
|Button Converged.As an anchor.default.chromium.png|469| Changed|
|Button Converged.As an anchor.hover.chromium.png|473| Changed|
|Button Converged.As an anchor.pressed.chromium.png|754| Changed|
|Button Converged.Circular and icon only - RTL.default.chromium.png|134| Changed|
|Button Converged.Circular and icon only - RTL.hover.chromium.png|348| Changed|
|Button Converged.Circular and icon only - RTL.pressed.chromium.png|411| Changed|
|Button Converged.Circular and icon only.default.chromium.png|90| Changed|
|Button Converged.Circular and icon only.hover.chromium.png|168| Changed|
|Button Converged.Circular and icon only.pressed.chromium.png|215| Changed|
|Button Converged.Circular.default.chromium.png|464| Changed|
|Button Converged.Circular.hover.chromium.png|469| Changed|
|Button Converged.Circular.pressed.chromium.png|723| Changed|
|Button Converged.Default - Dark Mode.default.chromium.png|50462| Changed|
|Button Converged.Default - Dark Mode.hover.chromium.png|50248| Changed|
|Button Converged.Default - Dark Mode.pressed.chromium.png|50558| Changed|
|Button Converged.Default - High Contrast.default.chromium.png|59948| Changed|
|Button Converged.Default - High Contrast.hover.chromium.png|58583| Changed|
|Button Converged.Default - High Contrast.pressed.chromium.png|58583| Changed|
|Button Converged.Default - RTL.default.chromium.png|563| Changed|
|Button Converged.Default - RTL.hover.chromium.png|582| Changed|
|Button Converged.Default - RTL.pressed.chromium.png|928| Changed|
|Button Converged.Default.default.chromium.png|469| Changed|
|Button Converged.Default.hover.chromium.png|474| Changed|
|Button Converged.Default.pressed.chromium.png|754| Changed|
|Button Converged.Disabled - Dark Mode.default.chromium.png|50851| Changed|
|Button Converged.Disabled - Dark Mode.hover.chromium.png|50851| Changed|
|Button Converged.Disabled - Dark Mode.pressed.chromium.png|50851| Changed|
|Button Converged.Disabled - High Contrast.default.chromium.png|60107| Changed|
|Button Converged.Disabled - High Contrast.hover.chromium.png|60107| Changed|
|Button Converged.Disabled - High Contrast.pressed.chromium.png|60107| Changed|
|Button Converged.Disabled.default.chromium.png|438| Changed|
|Button Converged.Disabled.hover.chromium.png|438| Changed|
|Button Converged.Disabled.pressed.chromium.png|438| Changed|
|Button Converged.Icon only.default.chromium.png|95| Changed|
|Button Converged.Icon only.hover.chromium.png|172| Changed|
|Button Converged.Icon only.pressed.chromium.png|243| Changed|
|Button Converged.Outline Disabled - Dark Mode.default.chromium.png|50636| Changed|
|Button Converged.Outline Disabled - Dark Mode.hover.chromium.png|50636| Changed|
|Button Converged.Outline Disabled - Dark Mode.pressed.chromium.png|50636| Changed|
|Button Converged.Outline Disabled - High Contrast.default.chromium.png|60107| Changed|
|Button Converged.Outline Disabled - High Contrast.hover.chromium.png|60107| Changed|
|Button Converged.Outline Disabled - High Contrast.pressed.chromium.png|60107| Changed|
|Button Converged.Outline Disabled.default.chromium.png|316| Changed|
|Button Converged.Outline Disabled.hover.chromium.png|316| Changed|
|Button Converged.Outline Disabled.pressed.chromium.png|316| Changed|
|Button Converged.Outline.default.chromium.png|469| Changed|
|Button Converged.Outline.hover.chromium.png|479| Changed|
|Button Converged.Outline.pressed.chromium.png|500| Changed|
|Button Converged.Primary - Dark Mode.default.chromium.png|50112| Changed|
|Button Converged.Primary - Dark Mode.hover.chromium.png|50000| Changed|
|Button Converged.Primary - Dark Mode.pressed.chromium.png|50437| Changed|
|Button Converged.Primary - High Contrast.default.chromium.png|57816| Changed|
|Button Converged.Primary - High Contrast.hover.chromium.png|58580| Changed|
|Button Converged.Primary - High Contrast.pressed.chromium.png|58580| Changed|
|Button Converged.Primary Disabled - Dark Mode.default.chromium.png|50898| Changed|
|Button Converged.Primary Disabled - Dark Mode.hover.chromium.png|50898| Changed|
|Button Converged.Primary Disabled - Dark Mode.pressed.chromium.png|50898| Changed|
|Button Converged.Primary Disabled - High Contrast.default.chromium.png|60278| Changed|
|Button Converged.Primary Disabled - High Contrast.hover.chromium.png|60278| Changed|
|Button Converged.Primary Disabled - High Contrast.pressed.chromium.png|60278| Changed|
|Button Converged.Primary Disabled.default.chromium.png|422| Changed|
|Button Converged.Primary Disabled.hover.chromium.png|422| Changed|
|Button Converged.Primary Disabled.pressed.chromium.png|422| Changed|
|Button Converged.Primary.default.chromium.png|1867| Changed|
|Button Converged.Primary.hover.chromium.png|1979| Changed|
|Button Converged.Primary.pressed.chromium.png|2304| Changed|
|Button Converged.Size large - with long text wrapping.chromium.png|2938| Changed|
|Button Converged.Size large.default.chromium.png|798| Changed|
|Button Converged.Size large.hover.chromium.png|1077| Changed|
|Button Converged.Size large.pressed.chromium.png|1625| Changed|
|Button Converged.Size medium - with long text wrapping.chromium.png|2474| Changed|
|Button Converged.Size small - with long text wrapping.chromium.png|1498| Changed|
|Button Converged.Size small.default.chromium.png|399| Changed|
|Button Converged.Size small.hover.chromium.png|500| Changed|
|Button Converged.Size small.pressed.chromium.png|718| Changed|
|Button Converged.Subtle - Dark Mode.default.chromium.png|50555| Changed|
|Button Converged.Subtle - Dark Mode.hover.chromium.png|50359| Changed|
|Button Converged.Subtle - Dark Mode.pressed.chromium.png|50469| Changed|
|Button Converged.Subtle - High Contrast.default.chromium.png|60209| Changed|
|Button Converged.Subtle - High Contrast.hover.chromium.png|58580| Changed|
|Button Converged.Subtle - High Contrast.pressed.chromium.png|58580| Changed|
|Button Converged.Subtle Disabled - Dark Mode.default.chromium.png|50662| Changed|
|Button Converged.Subtle Disabled - Dark Mode.hover.chromium.png|50662| Changed|
|Button Converged.Subtle Disabled - Dark Mode.pressed.chromium.png|50662| Changed|
|Button Converged.Subtle Disabled - High Contrast.default.chromium.png|60278| Changed|
|Button Converged.Subtle Disabled - High Contrast.hover.chromium.png|60278| Changed|
|Button Converged.Subtle Disabled - High Contrast.pressed.chromium.png|60278| Changed|
|Button Converged.Subtle Disabled.default.chromium.png|285| Changed|
|Button Converged.Subtle Disabled.hover.chromium.png|285| Changed|
|Button Converged.Subtle Disabled.pressed.chromium.png|285| Changed|
|Button Converged.Subtle.default.chromium.png|391| Changed|
|Button Converged.Subtle.hover.chromium.png|416| Changed|
|Button Converged.Subtle.pressed.chromium.png|709| Changed|
|Button Converged.Transparent - Dark Mode.default.chromium.png|50555| Changed|
|Button Converged.Transparent - Dark Mode.hover.chromium.png|50616| Changed|
|Button Converged.Transparent - Dark Mode.pressed.chromium.png|50639| Changed|
|Button Converged.Transparent - High Contrast.default.chromium.png|60209| Changed|
|Button Converged.Transparent - High Contrast.hover.chromium.png|58580| Changed|
|Button Converged.Transparent - High Contrast.pressed.chromium.png|58580| Changed|
|Button Converged.Transparent Disabled - Dark Mode.default.chromium.png|50662| Changed|
|Button Converged.Transparent Disabled - Dark Mode.hover.chromium.png|50662| Changed|
|Button Converged.Transparent Disabled - Dark Mode.pressed.chromium.png|50662| Changed|
|Button Converged.Transparent Disabled - High Contrast.default.chromium.png|60278| Changed|
|Button Converged.Transparent Disabled - High Contrast.hover.chromium.png|60278| Changed|
|Button Converged.Transparent Disabled - High Contrast.pressed.chromium.png|60278| Changed|
|Button Converged.Transparent Disabled.default.chromium.png|285| Changed|
|Button Converged.Transparent Disabled.hover.chromium.png|285| Changed|
|Button Converged.Transparent Disabled.pressed.chromium.png|285| Changed|
|Button Converged.Transparent.default.chromium.png|391| Changed|
|Button Converged.Transparent.hover.chromium.png|369| Changed|
|Button Converged.Transparent.pressed.chromium.png|379| Changed|
|Button Converged.With icon after content - RTL.default.chromium.png|692| Changed|
|Button Converged.With icon after content - RTL.hover.chromium.png|921| Changed|
|Button Converged.With icon after content - RTL.pressed.chromium.png|1344| Changed|
|Button Converged.With icon after content.default.chromium.png|614| Changed|
|Button Converged.With icon after content.hover.chromium.png|795| Changed|
|Button Converged.With icon after content.pressed.chromium.png|1115| Changed|
|Button Converged.With icon before content - RTL.default.chromium.png|692| Changed|
|Button Converged.With icon before content - RTL.hover.chromium.png|921| Changed|
|Button Converged.With icon before content - RTL.pressed.chromium.png|1344| Changed|
|Button Converged.With icon before content.default.chromium.png|608| Changed|
|Button Converged.With icon before content.hover.chromium.png|800| Changed|
|Button Converged.With icon before content.pressed.chromium.png|1120| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.click.chromium.png|35318| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.hover.chromium.png|31936| Changed|
|Card Converged - Interactive.appearance interactive - Filled - Dark Mode.normal.chromium.png|34177| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.click.chromium.png|20420| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.hover.chromium.png|20420| Changed|
|Card Converged - Interactive.appearance interactive - Filled - High Contrast.normal.chromium.png|39835| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.click.chromium.png|36582| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.hover.chromium.png|33050| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - Dark Mode.normal.chromium.png|35318| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.click.chromium.png|20420| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.hover.chromium.png|20420| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative - High Contrast.normal.chromium.png|39835| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.click.chromium.png|17257| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.hover.chromium.png|15320| Changed|
|Card Converged - Interactive.appearance interactive - Filled Alternative.normal.chromium.png|13796| Changed|
|Card Converged - Interactive.appearance interactive - Filled.click.chromium.png|16792| Changed|
|Card Converged - Interactive.appearance interactive - Filled.hover.chromium.png|13620| Changed|
|Card Converged - Interactive.appearance interactive - Filled.normal.chromium.png|13976| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.click.chromium.png|34421| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.hover.chromium.png|34458| Changed|
|Card Converged - Interactive.appearance interactive - Outline - Dark Mode.normal.chromium.png|34403| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|20124| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|20124| Changed|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|39835| Changed|
|Card Converged - Interactive.appearance interactive - Outline.click.chromium.png|13696| Changed|
|Card Converged - Interactive.appearance interactive - Outline.hover.chromium.png|13770| Changed|
|Card Converged - Interactive.appearance interactive - Outline.normal.chromium.png|13807| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|33613| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|32491| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|34177| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.click.chromium.png|20420| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.hover.chromium.png|20420| Changed|
|Card Converged - Interactive.appearance interactive - Subtle - High Contrast.normal.chromium.png|39835| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|16793| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|13621| Changed|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|13977| Changed|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|16792| Changed|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|13620| Changed|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|13976| Changed|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|15607| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.click.chromium.png|35318| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.hover.chromium.png|31936| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.normal.chromium.png|34177| Changed|
|Card Converged - Selectable.appearance selectable - Filled - Dark Mode.selected.chromium.png|32698| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.click.chromium.png|20420| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.hover.chromium.png|20420| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.normal.chromium.png|39835| Changed|
|Card Converged - Selectable.appearance selectable - Filled - High Contrast.selected.chromium.png|20124| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.click.chromium.png|17276| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.hover.chromium.png|14259| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.normal.chromium.png|14638| Changed|
|Card Converged - Selectable.appearance selectable - Filled - RTL.selected.chromium.png|16154| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.click.chromium.png|36582| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.hover.chromium.png|33050| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.normal.chromium.png|35318| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - Dark Mode.selected.chromium.png|33856| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.click.chromium.png|20420| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.hover.chromium.png|20420| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.normal.chromium.png|39835| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - High Contrast.selected.chromium.png|20124| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.click.chromium.png|17713| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.hover.chromium.png|15896| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.normal.chromium.png|14446| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative - RTL.selected.chromium.png|16603| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.click.chromium.png|17257| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.hover.chromium.png|15320| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.normal.chromium.png|13796| Changed|
|Card Converged - Selectable.appearance selectable - Filled Alternative.selected.chromium.png|16085| Changed|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|16792| Changed|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|13620| Changed|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|13976| Changed|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|15607| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.click.chromium.png|34421| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.hover.chromium.png|34458| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.normal.chromium.png|34403| Changed|
|Card Converged - Selectable.appearance selectable - Outline - Dark Mode.selected.chromium.png|34458| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.click.chromium.png|20124| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.hover.chromium.png|20124| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.normal.chromium.png|39835| Changed|
|Card Converged - Selectable.appearance selectable - Outline - High Contrast.selected.chromium.png|20124| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.click.chromium.png|14359| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.hover.chromium.png|14432| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.normal.chromium.png|14469| Changed|
|Card Converged - Selectable.appearance selectable - Outline - RTL.selected.chromium.png|14432| Changed|
|Card Converged - Selectable.appearance selectable - Outline.click.chromium.png|13696| Changed|
|Card Converged - Selectable.appearance selectable - Outline.hover.chromium.png|13770| Changed|
|Card Converged - Selectable.appearance selectable - Outline.normal.chromium.png|13807| Changed|
|Card Converged - Selectable.appearance selectable - Outline.selected.chromium.png|13770| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.click.chromium.png|33613| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.hover.chromium.png|32491| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.normal.chromium.png|34177| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - Dark Mode.selected.chromium.png|33276| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.click.chromium.png|20420| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.hover.chromium.png|20420| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.normal.chromium.png|39835| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - High Contrast.selected.chromium.png|20124| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.click.chromium.png|17277| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.hover.chromium.png|14260| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.normal.chromium.png|14639| Changed|
|Card Converged - Selectable.appearance selectable - Subtle - RTL.selected.chromium.png|16154| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.click.chromium.png|16793| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.hover.chromium.png|13621| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.normal.chromium.png|13977| Changed|
|Card Converged - Selectable.appearance selectable - Subtle.selected.chromium.png|15608| Changed|
|Card Converged.CardHeader.normal.chromium.png|27691| Changed|
|Card Converged.appearance - Dark Mode.normal.chromium.png|429220| Changed|
|Card Converged.appearance - High Contrast.normal.chromium.png|494199| Changed|
|Card Converged.appearance - RTL.normal.chromium.png|55465| Changed|
|Card Converged.appearance.normal.chromium.png|53246| Changed|
|Card Converged.card templates.normal.chromium.png|8363| Changed|
|Card Converged.orientation.normal.chromium.png|31031| Changed|
|Card Converged.size.normal.chromium.png|64445| Changed|
|CompoundButton Converged.Circular and icon only - RTL.default.chromium.png|497| Changed|
|CompoundButton Converged.Circular and icon only - RTL.hover.chromium.png|1336| Changed|
|CompoundButton Converged.Circular and icon only - RTL.pressed.chromium.png|1507| Changed|
|CompoundButton Converged.Circular and icon only.default.chromium.png|331| Changed|
|CompoundButton Converged.Circular and icon only.hover.chromium.png|527| Changed|
|CompoundButton Converged.Circular and icon only.pressed.chromium.png|649| Changed|
|CompoundButton Converged.Circular.default.chromium.png|970| Changed|
|CompoundButton Converged.Circular.hover.chromium.png|994| Changed|
|CompoundButton Converged.Circular.pressed.chromium.png|1954| Changed|
|CompoundButton Converged.Default - RTL.default.chromium.png|1345| Changed|
|CompoundButton Converged.Default - RTL.pressed.chromium.png|2659| Changed|
|CompoundButton Converged.Default.default.chromium.png|984| Changed|
|CompoundButton Converged.Default.pressed.chromium.png|2069| Changed|
|CompoundButton Converged.Disabled.default.chromium.png|1277| Changed|
|CompoundButton Converged.Disabled.hover.chromium.png|1277| Changed|
|CompoundButton Converged.Disabled.pressed.chromium.png|1277| Changed|
|CompoundButton Converged.Icon only.default.chromium.png|341| Changed|
|CompoundButton Converged.Icon only.hover.chromium.png|536| Changed|
|CompoundButton Converged.Icon only.pressed.chromium.png|729| Changed|
|CompoundButton Converged.Outline Disabled.default.chromium.png|795| Changed|
|CompoundButton Converged.Outline Disabled.hover.chromium.png|795| Changed|
|CompoundButton Converged.Outline Disabled.pressed.chromium.png|795| Changed|
|CompoundButton Converged.Outline.default.chromium.png|984| Changed|
|CompoundButton Converged.Outline.hover.chromium.png|1024| Changed|
|CompoundButton Converged.Outline.pressed.chromium.png|1059| Changed|
|CompoundButton Converged.Primary Disabled.default.chromium.png|1249| Changed|
|CompoundButton Converged.Primary Disabled.hover.chromium.png|1249| Changed|
|CompoundButton Converged.Primary Disabled.pressed.chromium.png|1249| Changed|
|CompoundButton Converged.Primary.default.chromium.png|6569| Changed|
|CompoundButton Converged.Primary.hover.chromium.png|6967| Changed|
|CompoundButton Converged.Primary.pressed.chromium.png|8108| Changed|
|CompoundButton Converged.Size large - with long text wrapping.chromium.png|3720| Changed|
|CompoundButton Converged.Size large.default.chromium.png|1665| Changed|
|CompoundButton Converged.Size large.hover.chromium.png|2079| Changed|
|CompoundButton Converged.Size large.pressed.chromium.png|4074| Changed|
|CompoundButton Converged.Size medium - with long text wrapping.chromium.png|3089| Changed|
|CompoundButton Converged.Size small - with long text wrapping.chromium.png|3040| Changed|
|CompoundButton Converged.Size small.default.chromium.png|1615| Changed|
|CompoundButton Converged.Size small.hover.chromium.png|2070| Changed|
|CompoundButton Converged.Size small.pressed.chromium.png|3126| Changed|
|CompoundButton Converged.Subtle Disabled.default.chromium.png|743| Changed|
|CompoundButton Converged.Subtle Disabled.hover.chromium.png|743| Changed|
|CompoundButton Converged.Subtle Disabled.pressed.chromium.png|743| Changed|
|CompoundButton Converged.Subtle.default.chromium.png|877| Changed|
|CompoundButton Converged.Subtle.hover.chromium.png|909| Changed|
|CompoundButton Converged.Subtle.pressed.chromium.png|1993| Changed|
|CompoundButton Converged.Transparent Disabled.default.chromium.png|743| Changed|
|CompoundButton Converged.Transparent Disabled.hover.chromium.png|743| Changed|
|CompoundButton Converged.Transparent Disabled.pressed.chromium.png|743| Changed|
|CompoundButton Converged.Transparent.default.chromium.png|877| Changed|
|CompoundButton Converged.Transparent.hover.chromium.png|843| Changed|
|CompoundButton Converged.Transparent.pressed.chromium.png|858| Changed|
|CompoundButton Converged.With icon after content - RTL.default.chromium.png|1840| Changed|
|CompoundButton Converged.With icon after content - RTL.pressed.chromium.png|4519| Changed|
|CompoundButton Converged.With icon after content.default.chromium.png|1422| Changed|
|CompoundButton Converged.With icon after content.pressed.chromium.png|3009| Changed|
|CompoundButton Converged.With icon before content - RTL.default.chromium.png|1840| Changed|
|CompoundButton Converged.With icon before content - RTL.pressed.chromium.png|4519| Changed|
|CompoundButton Converged.With icon before content.default.chromium.png|1436| Changed|
|CompoundButton Converged.With icon before content.pressed.chromium.png|3291| Changed|
|CounterBadge Converged - colors.default - RTL.chromium.png|1534| Changed|
|CounterBadge Converged - colors.default.chromium.png|948| Changed|
|Dialog.actions position start & position end - Dark Mode.chromium.png|78570| Changed|
|Dialog.actions position start & position end - High Contrast.chromium.png|100163| Changed|
|Dialog.actions position start & position end - RTL.chromium.png|47119| Changed|
|Dialog.actions position start & position end.chromium.png|46761| Changed|
|Dialog.actions position start - Dark Mode.chromium.png|78850| Changed|
|Dialog.actions position start - High Contrast.chromium.png|100500| Changed|
|Dialog.actions position start - RTL.chromium.png|47251| Changed|
|Dialog.actions position start.chromium.png|46753| Changed|
|Dialog.alert - Dark Mode.chromium.png|78825| Changed|
|Dialog.alert - High Contrast.chromium.png|100509| Changed|
|Dialog.alert - RTL.chromium.png|47250| Changed|
|Dialog.alert.chromium.png|46741| Changed|
|Dialog.default - Dark Mode.chromium.png|78825| Changed|
|Dialog.default - High Contrast.chromium.png|100509| Changed|
|Dialog.default - RTL.chromium.png|47250| Changed|
|Dialog.default.chromium.png|46741| Changed|
|Dialog.nested - Dark Mode.chromium.png|74611| Changed|
|Dialog.nested - High Contrast.chromium.png|93967| Changed|
|Dialog.nested - RTL.chromium.png|42374| Changed|
|Dialog.nested.chromium.png|42097| Changed|
|Dialog.no actions - Dark Mode.chromium.png|70653| Changed|
|Dialog.no actions - High Contrast.chromium.png|89254| Changed|
|Dialog.no actions - RTL.chromium.png|40174| Changed|
|Dialog.no actions.chromium.png|39932| Changed|
|Dialog.no title & no actions - Dark Mode.chromium.png|64107| Changed|
|Dialog.no title & no actions - High Contrast.chromium.png|80155| Changed|
|Dialog.no title & no actions - RTL.chromium.png|33044| Changed|
|Dialog.no title & no actions.chromium.png|32773| Changed|
|Dialog.no title - Dark Mode.chromium.png|70688| Changed|
|Dialog.no title - High Contrast.chromium.png|89637| Changed|
|Dialog.no title - RTL.chromium.png|40896| Changed|
|Dialog.no title.chromium.png|40579| Changed|
|Dialog.non-modal - Dark Mode.chromium.png|141056| Changed|
|Dialog.non-modal - High Contrast.chromium.png|163321| Changed|
|Dialog.non-modal - RTL.chromium.png|15557| Changed|
|Dialog.non-modal.chromium.png|14673| Changed|
|Dialog.scroll long content - Dark Mode.chromium.png|235569| Changed|
|Dialog.scroll long content - High Contrast.chromium.png|314966| Changed|
|Dialog.scroll long content - RTL.chromium.png|249463| Changed|
|Dialog.scroll long content.chromium.png|248858| Changed|
|Dialog.title custom action - Dark Mode.chromium.png|71174| Changed|
|Dialog.title custom action - High Contrast.chromium.png|90232| Changed|
|Dialog.title custom action - RTL.chromium.png|40698| Changed|
|Dialog.title custom action.chromium.png|40596| Changed|
|Image Converged.Image Fluid.normal.chromium.png|379| Changed|
|Image Converged.Image Shadow.normal.chromium.png|314| Changed|
|Image Converged.Image Shape.normal.chromium.png|4853| Changed|
|Image Converged.Image Variations Border.normal.chromium.png|11043| Changed|
|Image Converged.Image Variations Fallback.normal.chromium.png|712| Changed|
|MenuButton Converged.Circular and icon only.default.chromium.png|90| Changed|
|MenuButton Converged.Circular and icon only.hover.chromium.png|168| Changed|
|MenuButton Converged.Circular and icon only.pressed.chromium.png|215| Changed|
|MenuButton Converged.Circular.default.chromium.png|520| Changed|
|MenuButton Converged.Circular.hover.chromium.png|524| Changed|
|MenuButton Converged.Circular.pressed.chromium.png|830| Changed|
|MenuButton Converged.Default - RTL.default.chromium.png|582| Changed|
|MenuButton Converged.Default - RTL.hover.chromium.png|602| Changed|
|MenuButton Converged.Default - RTL.pressed.chromium.png|1008| Changed|
|MenuButton Converged.Default.default.chromium.png|527| Changed|
|MenuButton Converged.Default.hover.chromium.png|530| Changed|
|MenuButton Converged.Default.pressed.chromium.png|862| Changed|
|MenuButton Converged.Disabled.default.chromium.png|485| Changed|
|MenuButton Converged.Disabled.hover.chromium.png|485| Changed|
|MenuButton Converged.Disabled.pressed.chromium.png|485| Changed|
|MenuButton Converged.Icon only.default.chromium.png|95| Changed|
|MenuButton Converged.Icon only.hover.chromium.png|172| Changed|
|MenuButton Converged.Icon only.pressed.chromium.png|243| Changed|
|MenuButton Converged.Outline Disabled.default.chromium.png|341| Changed|
|MenuButton Converged.Outline Disabled.hover.chromium.png|341| Changed|
|MenuButton Converged.Outline Disabled.pressed.chromium.png|341| Changed|
|MenuButton Converged.Outline.default.chromium.png|527| Changed|
|MenuButton Converged.Outline.hover.chromium.png|538| Changed|
|MenuButton Converged.Outline.pressed.chromium.png|561| Changed|
|MenuButton Converged.Primary Disabled.default.chromium.png|467| Changed|
|MenuButton Converged.Primary Disabled.hover.chromium.png|467| Changed|
|MenuButton Converged.Primary Disabled.pressed.chromium.png|467| Changed|
|MenuButton Converged.Primary.default.chromium.png|2157| Changed|
|MenuButton Converged.Primary.hover.chromium.png|2288| Changed|
|MenuButton Converged.Primary.pressed.chromium.png|2669| Changed|
|MenuButton Converged.Size large - with long text wrapping.chromium.png|2981| Changed|
|MenuButton Converged.Size large.default.chromium.png|827| Changed|
|MenuButton Converged.Size large.hover.chromium.png|1106| Changed|
|MenuButton Converged.Size large.pressed.chromium.png|1748| Changed|
|MenuButton Converged.Size medium - with long text wrapping.chromium.png|2463| Changed|
|MenuButton Converged.Size small - with long text wrapping.chromium.png|1509| Changed|
|MenuButton Converged.Size small.default.chromium.png|413| Changed|
|MenuButton Converged.Size small.hover.chromium.png|514| Changed|
|MenuButton Converged.Size small.pressed.chromium.png|782| Changed|
|MenuButton Converged.Subtle Disabled.default.chromium.png|306| Changed|
|MenuButton Converged.Subtle Disabled.hover.chromium.png|306| Changed|
|MenuButton Converged.Subtle Disabled.pressed.chromium.png|306| Changed|
|MenuButton Converged.Subtle.default.chromium.png|439| Changed|
|MenuButton Converged.Subtle.hover.chromium.png|466| Changed|
|MenuButton Converged.Subtle.pressed.chromium.png|811| Changed|
|MenuButton Converged.Transparent Disabled.default.chromium.png|306| Changed|
|MenuButton Converged.Transparent Disabled.hover.chromium.png|306| Changed|
|MenuButton Converged.Transparent Disabled.pressed.chromium.png|306| Changed|
|MenuButton Converged.Transparent.default.chromium.png|439| Changed|
|MenuButton Converged.Transparent.hover.chromium.png|412| Changed|
|MenuButton Converged.Transparent.pressed.chromium.png|424| Changed|
|MenuButton Converged.With icon.default.chromium.png|623| Changed|
|MenuButton Converged.With icon.hover.chromium.png|816| Changed|
|MenuButton Converged.With icon.pressed.chromium.png|1192| Changed|
|Portal.Apply Class Names.should have green border.chromium.png|1056| Changed|
|ToggleButton Converged.Checked.default.chromium.png|633| Changed|
|ToggleButton Converged.Checked.hover.chromium.png|474| Changed|
|ToggleButton Converged.Checked.pressed.chromium.png|754| Changed|
|ToggleButton Converged.Circular and icon only.default.chromium.png|90| Changed|
|ToggleButton Converged.Circular and icon only.hover.chromium.png|168| Changed|
|ToggleButton Converged.Circular and icon only.pressed.chromium.png|215| Changed|
|ToggleButton Converged.Circular.default.chromium.png|464| Changed|
|ToggleButton Converged.Circular.hover.chromium.png|469| Changed|
|ToggleButton Converged.Circular.pressed.chromium.png|723| Changed|
|ToggleButton Converged.Default - Dark Mode.default.chromium.png|50462| Changed|
|ToggleButton Converged.Default - Dark Mode.hover.chromium.png|50248| Changed|
|ToggleButton Converged.Default - Dark Mode.pressed.chromium.png|50558| Changed|
|ToggleButton Converged.Default - High Contrast.default.chromium.png|59948| Changed|
|ToggleButton Converged.Default - High Contrast.hover.chromium.png|58583| Changed|
|ToggleButton Converged.Default - High Contrast.pressed.chromium.png|58583| Changed|
|ToggleButton Converged.Default - RTL.default.chromium.png|563| Changed|
|ToggleButton Converged.Default - RTL.hover.chromium.png|582| Changed|
|ToggleButton Converged.Default - RTL.pressed.chromium.png|928| Changed|
|ToggleButton Converged.Default.default.chromium.png|469| Changed|
|ToggleButton Converged.Default.hover.chromium.png|474| Changed|
|ToggleButton Converged.Default.pressed.chromium.png|754| Changed|
|ToggleButton Converged.Disabled.default.chromium.png|438| Changed|
|ToggleButton Converged.Disabled.hover.chromium.png|438| Changed|
|ToggleButton Converged.Disabled.pressed.chromium.png|438| Changed|
|ToggleButton Converged.Icon only.default.chromium.png|95| Changed|
|ToggleButton Converged.Icon only.hover.chromium.png|172| Changed|
|ToggleButton Converged.Icon only.pressed.chromium.png|243| Changed|
|ToggleButton Converged.Outline.default.chromium.png|469| Changed|
|ToggleButton Converged.Outline.hover.chromium.png|479| Changed|
|ToggleButton Converged.Outline.pressed.chromium.png|500| Changed|
|ToggleButton Converged.Primary - Dark Mode.default.chromium.png|50112| Changed|
|ToggleButton Converged.Primary - Dark Mode.hover.chromium.png|50000| Changed|
|ToggleButton Converged.Primary - Dark Mode.pressed.chromium.png|50437| Changed|
|ToggleButton Converged.Primary - High Contrast.default.chromium.png|57816| Changed|
|ToggleButton Converged.Primary - High Contrast.hover.chromium.png|58580| Changed|
|ToggleButton Converged.Primary - High Contrast.pressed.chromium.png|58580| Changed|
|ToggleButton Converged.Primary Checked.default.chromium.png|2077| Changed|
|ToggleButton Converged.Primary Checked.hover.chromium.png|1979| Changed|
|ToggleButton Converged.Primary Checked.pressed.chromium.png|2304| Changed|
|ToggleButton Converged.Primary Disabled.default.chromium.png|422| Changed|
|ToggleButton Converged.Primary Disabled.hover.chromium.png|422| Changed|
|ToggleButton Converged.Primary Disabled.pressed.chromium.png|422| Changed|
|ToggleButton Converged.Primary.default.chromium.png|1867| Changed|
|ToggleButton Converged.Primary.hover.chromium.png|1979| Changed|
|ToggleButton Converged.Primary.pressed.chromium.png|2304| Changed|
|ToggleButton Converged.Size large - with long text wrapping.chromium.png|2938| Changed|
|ToggleButton Converged.Size large.default.chromium.png|798| Changed|
|ToggleButton Converged.Size large.hover.chromium.png|1077| Changed|
|ToggleButton Converged.Size large.pressed.chromium.png|1625| Changed|
|ToggleButton Converged.Size medium - with long text wrapping.chromium.png|2474| Changed|
|ToggleButton Converged.Size small - with long text wrapping.chromium.png|1498| Changed|
|ToggleButton Converged.Size small.default.chromium.png|399| Changed|
|ToggleButton Converged.Size small.hover.chromium.png|500| Changed|
|ToggleButton Converged.Size small.pressed.chromium.png|718| Changed|
|ToggleButton Converged.Subtle Checked.default.chromium.png|606| Changed|
|ToggleButton Converged.Subtle Checked.hover.chromium.png|416| Changed|
|ToggleButton Converged.Subtle Checked.pressed.chromium.png|709| Changed|
|ToggleButton Converged.Subtle Disabled.default.chromium.png|285| Changed|
|ToggleButton Converged.Subtle Disabled.hover.chromium.png|285| Changed|
|ToggleButton Converged.Subtle Disabled.pressed.chromium.png|285| Changed|
|ToggleButton Converged.Subtle.default.chromium.png|391| Changed|
|ToggleButton Converged.Subtle.hover.chromium.png|416| Changed|
|ToggleButton Converged.Subtle.pressed.chromium.png|709| Changed|
|ToggleButton Converged.Transparent Checked.default.chromium.png|369| Changed|
|ToggleButton Converged.Transparent Checked.hover.chromium.png|369| Changed|
|ToggleButton Converged.Transparent Checked.pressed.chromium.png|379| Changed|
|ToggleButton Converged.Transparent Disabled.default.chromium.png|285| Changed|
|ToggleButton Converged.Transparent Disabled.hover.chromium.png|285| Changed|
|ToggleButton Converged.Transparent Disabled.pressed.chromium.png|285| Changed|
|ToggleButton Converged.Transparent.default.chromium.png|391| Changed|
|ToggleButton Converged.Transparent.hover.chromium.png|369| Changed|
|ToggleButton Converged.Transparent.pressed.chromium.png|379| Changed|
|ToggleButton Converged.With icon after content.default.chromium.png|614| Changed|
|ToggleButton Converged.With icon after content.hover.chromium.png|795| Changed|
|ToggleButton Converged.With icon after content.pressed.chromium.png|1115| Changed|
|ToggleButton Converged.With icon before content.default.chromium.png|608| Changed|
|ToggleButton Converged.With icon before content.hover.chromium.png|800| Changed|
|ToggleButton Converged.With icon before content.pressed.chromium.png|1120| Changed|
|Toolbar Converged.Default.Button Pressed.chromium.png|927| Changed|
|Toolbar Converged.Default.Toggle On.chromium.png|731| Changed|
|Toolbar Converged.Default.default.chromium.png|728| Changed|
|Toolbar Converged.Far Group.chromium.png|2576| Changed|
|Toolbar Converged.Large.Button Pressed.chromium.png|985| Changed|
|Toolbar Converged.Large.Toggle On.chromium.png|743| Changed|
|Toolbar Converged.Large.default.chromium.png|739| Changed|
|Toolbar Converged.Small.Button Pressed.chromium.png|918| Changed|
|Toolbar Converged.Small.Toggle On.chromium.png|730| Changed|
|Toolbar Converged.Small.default.chromium.png|727| Changed|
|Toolbar Converged.Vertical Button.Button Pressed.chromium.png|4002| Changed|
|Toolbar Converged.Vertical Button.Toggle On.chromium.png|3875| Changed|
|Toolbar Converged.Vertical Button.default.chromium.png|3871| Changed|
|Toolbar Converged.Vertical.Button Pressed.chromium.png|23576| Changed|
|Toolbar Converged.Vertical.Toggle On.chromium.png|23460| Changed|
|Toolbar Converged.Vertical.default.chromium.png|23494| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>